syntax = "proto3";

package model.messages.gradido;

import "Transfer.proto";
import "StateCreateGroup.proto";
import "StateGroupChangeParent.proto";
import "TransactionCreation.proto";

message TransactionBody {
	string memo = 1; // max 150 chars
	oneof data {
		StateCreateGroup createGroup = 2;
		StateGroupChangeParent groupChangeParent = 3;
		Transfer transfer = 4;
		TransactionCreation creation = 5;
	}
}
